src/diagram/utils/validate.ts

```diff
@@ -1,2 +1,6 @@
 export const kMaxNameCharacters = 30;
 export const kMaxNotesCharacters = 255;
+
+export const validNumber = (v: string) => !isNaN(+v);
```

Comment:
I would name this
```suggestion
export const isValidNumber = (v: string) => !isNaN(+v);
```
to make it clear that it's returning a boolean rather than converting to a valid number, for instance.